ACE/ace/OS_NS_sys_socket.inl

```diff
@@ -135,6 +135,9 @@ ACE_OS::closesocket (ACE_HANDLE handle)
   //       process.
 
   ACE_SOCKCALL_RETURN (::closesocket ((SOCKET) handle), int, -1);
+#elif defined ACE_LACKS_CLOSE
+  ACE_UNUSED_ARG (handle);
```

Comment:
This looks to be a wrong check, ACE_LACKS_CLOSESOCKET?

Reply:
I think it's correct -- `close()` is called if `ACE_LACKS_CLOSE` is not defined.

Reply:
Why not call ACE_OS::close here?

Reply:
That sounds good.